### DISCUSSION TITLE
Is it possible to apply user-defined loads in Anura3D in some way like the DLOAD in Abaqus?

### DISCUSSION THREAD

**Discussion by hkyang100** [Anura3D/Anura3D_OpenSource#6] [Q&A] [answered]
~~

**Comment by murphy-john** [comment]
Hello @hkyang100,
Could you provide additional detail about what you'd like to do with the user defined loads? There are methods to apply loads to nodes and material points (MPs) in GiD. You can control the load application as linear or stepwise at each loadstep. There are some examples of this in the tutorial manual examples.

**Reply by hkyang100** [reply]
John Murphy, thanks very much!
   I would like to apply the wave-induced hydrodynamic pressure to seabed soils. It involves the surface traction of fluid phases that varies with time and space. I am not sure if Anura3D-GID can apply this pressure, e.g. p=A*cos(k*x-omega*t)?

**Reply by FrancescaCeccato** [reply, marked as answer]
This is not implemented in the code now, however you can do it yourself by modifing the variable CalParams%Multipliers%WaterACurrent inside the subroutine UpdateMultipliersForLoadStep.

**Reply by hkyang100** [reply]
I got it. Thanks a lot!
